torchci/pages/nightlies.tsx

```diff
@@ -76,16 +75,16 @@ function ValidationRedPanel({
   channel,
   query_type,
 }: {
-  params: RocksetParam[];
+  params: { [key: string]: any };
   channel: string;
   query_type: string;
 }) {
   const url =
-    `/api/query/nightlies/` +
+    `/api/clickhouse/` +
     query_type +
     `_jobs_red?parameters=${encodeURIComponent(
       JSON.stringify([
-        ...params,
+        params,
```

Comment:
The ClickHouse params should be a dictionary/object of "fieldname": "value"
ex timezone: Intl.DateTimeFormat().resolvedOptions().timeZone etc

Reply:
A dictionary/object, no arrays

So this one would just be JSON.stringify({...params, channel})